<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-C3OONP8" failures="1" tests="2" name="Test" time="59.384" errors="0" timestamp="2023-12-31T16:11:11 IST">
  <testcase classname="contactsTests.CreateContactTest" name="createcontact" time="15.309">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at contactsTests.CreateContactTest.createcontact(CreateContactTest.java:37)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- createcontact -->
  <testcase classname="contactsTests.CreateContactTestWithLeadSourceTest" name="createNewContactwithLeadSourceValue" time="12.967"/>
  <testcase classname="generalUtilities.BaseClass" name="@AfterSuite asConfig" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite asConfig -->
  <testcase classname="generalUtilities.BaseClass" name="@BeforeSuite bsConfig" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite bsConfig -->
</testsuite> <!-- Test -->
